nbviewer/providers/__init__.py

```diff
@@ -21,7 +21,22 @@ def provider_handlers(providers):
        example, custom URLs which should be intercepted before being
        handed to the basic `url` handler
     """
-    return _load_provider_feature('default_handlers', providers)
+    # `handler_kwargs` is a dict of dicts: first dict is `handler_names`, which
+    # specifies the handler_classes to load for the providers, the second
+    # is `handler_settings` (see comments in `format_handlers` in nbviewer/handlers.py)
```

Comment:
This comment block would make a good docstring describing the `handler_kwargs` parameter.

Reply:
Ditto the above!